### ISSUE TITLE
UnityEngine.AndroidJavaException: java.lang.ClassNotFoundException: io.sentry.Sentry

### ISSUE THREAD

**Issue by oddgames-david** [getsentry/sentry-unity#1358]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which version of the SDK?

1.4.0

How did you install the package?

Git UPM

Which version of Unity?

2022.3.0f1

### Steps to Reproduce

Empty project
Add sentry via UPM
Build and run on device (android 13, samsung s22)

### Expected Result

It works

### Actual Result

This is reported to sentry as an issue (so at least it reports):

```
UnityEngine.AndroidJavaException: java.lang.ClassNotFoundException: io.sentry.Sentry
  Module "UnityEngine.AndroidJNISafe", in CheckException
  Module "UnityEngine.AndroidJNISafe", in FindClass
  Module "UnityEngine.AndroidJavaClass", in _AndroidJavaClass
  Module "UnityEngine.AndroidJavaClass", in .ctor
  Module "Sentry.Unity.Android.SentryJava", in GetSentryJava
  Module "Sentry.Unity.Android.SentryJava", in GetInstallationId
  Module "Sentry.Unity.Android.SentryNativeAndroid", in Configure
  Module "Sentry.Unity.SentryInitialization", in Init
```

**Comment by oddgames-david** [comment]
Maybe related to #1355

**Comment by bitsandfoxes** [comment]
> Probably caused by #1355?

Hey @oddgames-david and thanks for reporting this. That sounds likely. The PR updates adding Sentry Android to the "new" template file.

**Comment by oddgames-david** [comment]
Patch fix coming tonight?

**Comment by eefan000** [comment]
I also had the same problem.
https://feamber.sentry.io/share/issue/64a77a3a54b34408b2b66cb4c9c10fa5/

**Comment by bitsandfoxes** [comment]
I'm on it.

**Comment by bitsandfoxes** [comment]
Sorry for the wait. This should be fixed in `1.5.1`.
